python/lsst/pipe/tasks/postprocess.py

```diff
@@ -673,6 +673,9 @@ def addCalibColumns(self, catalog, exposure, exposureIdInfo, **kwargs):
         newCat = afwTable.SourceCatalog(schema)
         newCat.extend(catalog, mapper=mapper)
 
+        if self.config.doReevaluateSkyWcs:
+            afwTable.updateSourceCoords(exposure.wcs, newCat)
```

Comment:
Do you need to have a corresponding statement for `self.config.doReevaluatePhotoCalib`?